src/seq_mv/csr_matop_device.c

```diff
@@ -1139,5 +1139,15 @@ hypre_SortCSRCusparse( HYPRE_Int      n,
    hypre_TFree(pBuffer, HYPRE_MEMORY_DEVICE);
    HYPRE_CUSPARSE_CALL(cusparseDestroyCsru2csrInfo(sortInfoA));
 }
+
+HYPRE_Int
+hypre_CSRMatrixSortRow(hypre_CSRMatrix *A)
```

Comment:
@Hitenze Please check the interface change of this function. I am not sure why you wanted `hypre_CSRMatrix **Ap` as the input before.

Reply:
We should change to the new interface. The old interface is for test purposes and is no longer needed. Thank you!